docs(completion): incompatibility (#530)
pnpm/pnpm.io: docs/completion.md

title: Command line tab-completion
---

:::info

Completion for pnpm v9 is incompatible with completion for older pnpm versions.

> If you already install pnpm completion for version older than v9, you must uninstall it first so that completion for v9 may work properly.

Reviewer: If you wanted to split on paragraphs, you need to add a newline between them

Author: I don't actually want to create multiple paragraphs. I just want to source code to look nicer in the code editor.

> Completion for pnpm v9 is incompatible with completion for older pnpm versions.
> If you already install pnpm completion for version older than v9, you must uninstall it first so that completion for v9 may work properly.
> You may do so by removing section of code that contains `__tabtab` in your dot files.

Reviewer: ```suggestion
Completion for pnpm v9 is incompatible with completion for older pnpm versions.

If you have already installed pnpm completion for a version older than v9, you must uninstall it first to ensure that completion for v9 works properly.

You can do this by removing the section of code that contains `__tabtab` in your dot files.
```

I did not check the content of the message, it's just a small suggestion to improve grammar and readability.

title: Command line tab-completion
---

:::info

> Completion for pnpm v9 is incompatible with completion for older pnpm versions.
> 
> If you have already installed pnpm completion for a version older than v9, you must uninstall it first to ensure that completion for v9 works properly.
> 
> You can do this by removing the section of code that contains `__tabtab` in your dot files.

Reviewer 2: ```suggestion
Completion for pnpm v9 is incompatible with completion for older pnpm versions.
If you have already installed pnpm completion for a version older than v9, you must uninstall it first to ensure that completion for v9 works properly.
You can do this by removing the section of code that contains `__tabtab` in your dot files.
```
